projects/Mallard/src/screens/article/slider/SliderHeaderLowEnd.tsx

```diff
@@ -73,7 +73,14 @@ const SliderHeaderLowEnd = withNavigation(
                 >
                     <SliderBarWrapper
                         sections={sections}
-                        sliderPosition={sliderPosition}
+                        sliderPosition={
+                            Platform.OS === 'android'
```

Comment:
We have code like this: `Platform.OS === 'android'` in lots of places throughout the codebase. Do you think we can have some simple functions in one place we can use instead. For example:

```
const isAndroid: boolean = () => Platform.OS === 'android'
const isIos: boolean = () => Platform.OS === 'ios'
```

Reply:
Could do. We can also use React Native platform suffix (e.g. `Slider.android.tsx`) to help with making things a bit more readable.

There is a lot to refactor here